.teamcity/_self/projects/WebApp.kt

```diff
@@ -49,6 +49,28 @@ object BuildDockerImage : BuildType({
 	}
 
 	steps {
+
+		script {
+			name = "Webhook Start"
```

Comment:
Why do we need to run this webhook even before we build the image?

Reply:
IRC messaging.